core/src/main/java/com/linecorp/armeria/common/MediaTypeNames.java

```diff
@@ -0,0 +1,472 @@
+    /**
+     * {@value #WEBM_VIDEO}.
+     */
+    public static final String WEBM_VIDEO = "video/webm";
+    /**
+     * {@value #WMV}.
+     */
+    public static final String WMV = "video/x-ms-wmv";
+    /**
+     * {@value #FLV_VIDEO}.
+     */
+    public static final String FLV_VIDEO = "video/x-flv";
+    /**
+     * {@value #THREE_GPP_VIDEO}.
+     */
+    public static final String THREE_GPP_VIDEO = "video/3gpp";
+    /**
+     * {@value #THREE_GPP2_VIDEO}.
+     */
+    public static final String THREE_GPP2_VIDEO = "video/3gpp2";
+
+    /* application types */
+
+    /**
+     * {@value #APPLICATION_XML_UTF_8}.
+     */
+    public static final String APPLICATION_XML_UTF_8 = "application/xml; charset=utf-8";
+    /**
+     * {@value #ATOM_UTF_8}.
+     */
+    public static final String ATOM_UTF_8 = "application/atom+xml; charset=utf-8";
+    /**
+     * {@value #BZIP2}.
+     */
+    public static final String BZIP2 = "application/x-bzip2";
+    /**
+     * {@value #DART_UTF_8}.
+     */
+    public static final String DART_UTF_8 = "application/dart; charset=utf-8";
+    /**
+     * {@value #APPLE_PASSBOOK}.
+     */
+    public static final String APPLE_PASSBOOK = "application/vnd.apple.pkpass";
+    /**
+     * {@value #EOT}.
+     */
+    public static final String EOT = "application/vnd.ms-fontobject";
+    /**
+     * {@value #EPUB}.
+     */
+    public static final String EPUB = "application/epub+zip";
+    /**
+     * {@value #FORM_DATA}.
+     */
+    public static final String FORM_DATA = "application/x-www-form-urlencoded";
+    /**
+     * {@value #KEY_ARCHIVE}.
+     */
+    public static final String KEY_ARCHIVE = "application/pkcs12";
+    /**
+     * {@value #APPLICATION_BINARY}.
+     */
+    public static final String APPLICATION_BINARY = "application/binary";
+    /**
+     * {@value #GEO_JSON}.
+     */
+    public static final String GEO_JSON = "application/geo+json";
+    /**
+     * {@value #GZIP}.
+     */
+    public static final String GZIP = "application/x-gzip";
+    /**
+     * {@value #HAL_JSON}.
+     */
+    public static final String HAL_JSON = "application/hal+json";
+    /**
+     * {@value #JAVASCRIPT_UTF_8}.
+     */
+    public static final String JAVASCRIPT_UTF_8 = "application/javascript; charset=utf-8";
+    /**
+     * {@value #JOSE}.
+     */
+    public static final String JOSE = "application/jose";
+    /**
+     * {@value #JOSE_JSON}.
+     */
+    public static final String JOSE_JSON = "application/jose+json";
+    /**
+     * {@value #JSON_UTF_8}.
+     */
+    public static final String JSON_UTF_8 = "application/json; charset=utf-8";
+    /**
+     * {@value #JSON}.
+     */
+    public static final String JSON = "application/json";
+    /**
+     * {@value #JSON_PATCH}.
+     */
+    public static final String JSON_PATCH = "application/json-patch+json";
+    /**
+     * {@value #JSON_SEQ}.
+     */
+    public static final String JSON_SEQ = "application/json-seq";
```

Comment:
Could you migrate `@Produces("application/json-seq")` to `@Produces(JSON_SEQ)` and other similar things in our code base?

Reply:
I applied them all but test code (I am not sure applying test code since some are applied some are not. It seems weird to me).